applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -1846,13 +1846,10 @@ void BaseSolidElement::CalculateDampingMatrixWithLumpedMass(
 
     // 2.-Calculate mass matrix:
     if (alpha > std::numeric_limits<double>::epsilon()) {
-        MatrixType mass_matrix = ZeroMatrix( mat_size, mat_size );
         VectorType temp_vector(mat_size);
         CalculateLumpedMassVector(temp_vector);
         for (IndexType i = 0; i < mat_size; ++i)
-            mass_matrix(i, i) = temp_vector[i];
-
-        noalias( rDampingMatrix ) += alpha * mass_matrix;
+            rDampingMatrix(i, i) += alpha * temp_vector[i];
```

Comment:
Shouldn’t you also initialize the dampingmatrix to zero first?

Note to myself, we should also port this to be used by the other elements in the same fashion

Reply:
It is initialized some lines up

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/69c59dd82bed1c0f37ee5b177e6e78b9c4ea20ac/applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp#L1829